### ISSUE TITLE
Text color issues in some apps

### ISSUE THREAD

**Issue by sidewinda** [catppuccin/gtk#12]
I just noticed some issues with the text colors in FieZilla and Thunderbird.

It's always on the text for an selected item. In FileZilla the folders have white text. When an folder is selected the row gets a lighter purple background but the text turns into black or the darkst purple color of the theme whicht is not readable.

On Thunderbirds it's almost the same issue. The selected mailbox folder in the left sidebar also gets this dark text and is not readable.

I switched back to the original theme from Manjaro to test it and there these problem don't exist.

Any solution for this?

**Comment by auvalice** [comment]
The GTK engine determines the theme to be a light theme. The application then uses their own light/dark colors to render the text. Similar issues occur in Pop!OS . 
![image](https://user-images.githubusercontent.com/88582996/166109112-4308969c-288e-4d9b-9281-e0094fa1f26a.png)

The theme switcher thinks the light version is selected. Unfortunately I cannot switch back to Dark via this window; it'll instruct Pop to set the GTK theme to Pop!dark rather than Catppuccin. 
![image](https://user-images.githubusercontent.com/88582996/166109134-43d163e1-ad2b-4d0c-9145-1d6ea04a6ebb.png)

Themes in /usr/share/themes are defined as such:
![image](https://user-images.githubusercontent.com/88582996/166109206-ad3d5f52-893a-4316-94aa-62c443160f9c.png)

 I renamed the themes directories as follows:
`Catppuccin-{color}` => `Catppuccin-{color}-dark`
`Catppuccin-{color}-hdpi` => `Catppuccin-{color}-dark-hdpi`
`Catppuccin-{color}-xhdpi` => `Catppuccin-{color}-dark-xhdpi`

The result:
![image](https://user-images.githubusercontent.com/88582996/166109315-3a17e0ac-9c2c-41aa-9ddb-b21349234341.png)

(light/dark theme should probably not be determined based on the name of the folder, and I thought there were themes that didn't do this... but this works, so I am happy.)

**Comment by pedropalmav** [comment]
> The GTK engine determines the theme to be a light theme. The application then uses their own light/dark colors to render the text. Similar issues occur in Pop!OS . ![image](https://user-images.githubusercontent.com/88582996/166109112-4308969c-288e-4d9b-9281-e0094fa1f26a.png)
> 
> Themes in /usr/share/themes are defined as such: ![image](https://user-images.githubusercontent.com/88582996/166109206-ad3d5f52-893a-4316-94aa-62c443160f9c.png)
> 
> Catppuccin is missing this `-dark` suffix. I renamed the themes in my .themes directory as follows: `Catppuccin-{color}` => `Catppuccin-{color}-dark` `Catppuccin-{color}-hdpi` => `Catppuccin-{color}-dark-hdpi` `Catppuccin-{color}-xhdpi` => `Catppuccin-{color}-dark-xhdpi`
> 
> The result: ![image](https://user-images.githubusercontent.com/88582996/166109315-3a17e0ac-9c2c-41aa-9ddb-b21349234341.png)
> 
> (light/dark theme should probably not be determined based on the name of the folder, and I thought there were themes that didn't do this... but this works, so I am happy.)

I followed this steps in Pop! os 22.04 LTS and it worked partially. When i open the menu on the top bar i can't really see because the color of the selection is too bright. 
![image](https://user-images.githubusercontent.com/80769811/171004586-3e7be392-1169-49ca-be24-06735134b45b.png)

Moreover, now all the gnome shell asociated windows don't use the catppuccin color scheme. I tried to fix it by following this [issue](https://github.com/catppuccin/gtk/issues/13), but it didn´t work at all.

Does anyone knows how to solve this?

**Comment by eclairevoyant** [comment]
I'm not a contributor atm, but I don't think this is an issue with the theme. I'd recommend checking your GTK config files to ensure they're consistent.
For example, this is my config for GTK3. This is located at `~/.config/gtk-3.0/settings.ini` (I had to create the file). This seems to consistently produce a dark theme in the apps I used.
```ini
[Settings]
gtk-theme-name = Catppuccin-pink
gtk-application-prefer-dark-theme = true
```

GTK4 apps require different config, see https://wiki.archlinux.org/title/GTK#Dark_theme_variant
